misc/example_crud_blueprint.md

```
@@ -0,0 +1,207 @@
+    use: 240
+    verify: 180
+
+agent_defaults:
+  agent_role: implementer
+  required_tools: []
+  skills: ["python", "fastapi", "sqlite", "pytest"]
+---
+
+# Team Tasks API
+
+## Overview
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)